R/calcSteel_Projections.R

```diff
@@ -60,6 +60,12 @@ calcSteel_Projections <- function(subtype = 'production',
     }
   }
 
+  if (  do_use_expert_guess
+      & match.steel.estimates != 'IEA_ETP'
+      & "SSP2IndiaHigh" %in% scenarios){
```

Comment:
Should be `&&`.  (`&` is elementwise comparison.  Not relevant here, but we are teaching you R.  And a linter will complain, in case anybody cares about that.)